docs/docs/reference/functions/middleware-and-helpers.md

```diff
@@ -51,7 +51,41 @@ export default async function corsHandler(req, res) {
 
 ## Custom body parsing
 
-Generally useful for file upload support.
+Generally useful for file upload support or raw bodies.
 
-This is not yet supported. [Add a comment in the discussion if this is an
-important use case for you](https://github.com/gatsbyjs/gatsby/discussions/30735).
+Each function can export an object named `config`. This object will allow you to control the `body-parser` middleware used by Gatsby Functions. The `limit` property will allow configuration of payload size up to **_32mb_**.
+
+### Examples
+
+```js
+// raw body (string)
+export const config = {
+  bodyParser: {
+    text: {
+      type: "*/*",
+    },
+  },
+}
+```
+
+```js
+// raw body (buffer)
+export const config = {
+  bodyParser: {
+    raw: {
+      type: "*/*",
```

Comment:
what does the `type` mean?

Reply:
type is body-parser's name for content-type. I will add more documentation for the options.

Reply:
Could we use `contentType` and default to all? This doesn't seem like something most people will care about.

Reply:
It has a default and isn't required. this example explains how to get the raw text from a request.

Reply:
> Could we use contentType and default to all? This doesn't seem like something most people will care about.

All `body-parser` middlewares (by default) try to target specific (content)type:
 - `.raw()` -> `application/octet-stream`
 - `.text()` -> `text/plain`
 - `.json()` -> `application/json`
 - `.urlencoded()` -> `application/x-www-form-urlencoded`

So the resulting `req.body` is represented by appropiate type.

This works well enough for most cases, but sometimes users do need access to "raw buffer" (despite technically response being for example `application/json`) to verify signatures on the funciton handler side (verify that request is valid and is actually produced by 3rd party system - for example Stripe - see Node.js example in https://stripe.com/docs/webhooks/signatures#verify-official-libraries, where they set `.raw()` middleware for `application/json`) which won't work with our defaults